iceoryx_posh/include/iceoryx_posh/internal/popo/base_server.hpp

```diff
@@ -98,7 +98,7 @@ class BaseServer
     friend class NotificationAttorney;
 
   protected:
-    using SelfType = BaseServer<PortT>;
+    using SelfType = BaseServer<PortT, TriggerHandleT>;
```

Comment:
My eye tends to miss the last letter. What do you think about?

```suggestion
    using SelfType = BaseServer<PortType, TriggerHandleType>;
```

Reply:
`PortType2`? Just kidding :stuck_out_tongue: 

Just remove the alias, then the name should be the same.

Reply:
I'm not sure if it breaks something. How about `TPort` and `TTriggerHandle`?

Reply:
Or just `P` and `T` :stuck_out_tongue_winking_eye:

Reply:
Not too convincing :P Why not try removing the alias and see if this breaks something? ;)

Reply:
@mossmaurice because it breaks `using RequestSampleDeleter = RequestDeleter<typename BaseClientT::PortType>;` and other stuff

Considering we are so close to release, there is still much to do and the template Names were not introduced in this PR would you agree to leave it as is?